--- gnome-applets-2.21.4/m4/system-tools-backends.m4~	2007-11-27 09:48:47.000000000 +0100
+++ gnome-applets-2.21.4/m4/system-tools-backends.m4	2008-01-16 12:31:33.795736049 +0100
@@ -7,7 +7,7 @@
 AC_DEFUN([AM_PATH_SYSTEM_TOOLS_BACKENDS],
 [
   AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
-  stb="system-tools-backends"
+  stb="system-tools-backends-2.0"
 
   if test x$PKG_CONFIG != xno ; then
     if $PKG_CONFIG --atleast-pkgconfig-version 0.12 ; then
