diff -Nur gnome-applets-1.100.0/battstat/Makefile.am gnome-applets-1.100.0.new/battstat/Makefile.am
--- gnome-applets-1.100.0/battstat/Makefile.am	Sun Apr 28 19:25:58 2002
+++ gnome-applets-1.100.0.new/battstat/Makefile.am	Sun Apr 28 23:04:24 2002
@@ -49,8 +49,6 @@
 schemadir   = @GCONF_SCHEMA_FILE_DIR@
 schema_DATA = battstat.schemas
 
-install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA) ;
 
 serverdir       = $(libdir)/bonobo/servers
 server_in_files = GNOME_BattstatApplet.server.in
diff -Nur gnome-applets-1.100.0/cdplayer/Makefile.am gnome-applets-1.100.0.new/cdplayer/Makefile.am
--- gnome-applets-1.100.0/cdplayer/Makefile.am	Sun Apr 28 19:26:00 2002
+++ gnome-applets-1.100.0.new/cdplayer/Makefile.am	Sun Apr 28 23:04:14 2002
@@ -54,6 +54,4 @@
 $(schema_DATA)	\
 $(ui_DATA)
 
-install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA) ;
 
diff -Nur gnome-applets-1.100.0/charpick/Makefile.am gnome-applets-1.100.0.new/charpick/Makefile.am
--- gnome-applets-1.100.0/charpick/Makefile.am	Sun Apr 28 19:26:03 2002
+++ gnome-applets-1.100.0.new/charpick/Makefile.am	Sun Apr 28 23:04:06 2002
@@ -34,6 +34,4 @@
 	$(ui_DATA)	\
 	$(schema_DATA)
 
-install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA) ;
 
diff -Nur gnome-applets-1.100.0/drivemount/Makefile.am gnome-applets-1.100.0.new/drivemount/Makefile.am
--- gnome-applets-1.100.0/drivemount/Makefile.am	Sun Apr 28 19:26:05 2002
+++ gnome-applets-1.100.0.new/drivemount/Makefile.am	Sun Apr 28 23:03:56 2002
@@ -59,6 +59,4 @@
 $(ui_DATA)	\
 drivemount-applet.png
 
-install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA) ;
 
diff -Nur gnome-applets-1.100.0/geyes/Makefile.am gnome-applets-1.100.0.new/geyes/Makefile.am
--- gnome-applets-1.100.0/geyes/Makefile.am	Sun Apr 28 19:26:07 2002
+++ gnome-applets-1.100.0.new/geyes/Makefile.am	Sun Apr 28 23:03:43 2002
@@ -44,7 +44,5 @@
 	     $(schema_DATA) \
 	     $(ui_DATA)
 
-install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA) ;
 
 
diff -Nur gnome-applets-1.100.0/gkb-new/Makefile.am gnome-applets-1.100.0.new/gkb-new/Makefile.am
--- gnome-applets-1.100.0/gkb-new/Makefile.am	Sun Apr 28 19:26:09 2002
+++ gnome-applets-1.100.0.new/gkb-new/Makefile.am	Sun Apr 28 23:03:33 2002
@@ -59,6 +59,4 @@
 	< gkb_xmmap.in >gkb_xmmap.tmp \
 	&& mv gkb_xmmap.tmp gkb_xmmap
 
-install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA) ;
 
diff -Nur gnome-applets-1.100.0/gtik/Makefile.am gnome-applets-1.100.0.new/gtik/Makefile.am
--- gnome-applets-1.100.0/gtik/Makefile.am	Sun Apr 28 19:26:17 2002
+++ gnome-applets-1.100.0.new/gtik/Makefile.am	Sun Apr 28 23:03:19 2002
@@ -34,6 +34,4 @@
 	$(ui_DATA)		\
 	$(schema_DATA)
 	
-install-data-local:
-		GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA) ;
 
diff -Nur gnome-applets-1.100.0/gweather/Makefile.am gnome-applets-1.100.0.new/gweather/Makefile.am
--- gnome-applets-1.100.0/gweather/Makefile.am	Sun Apr 28 19:26:20 2002
+++ gnome-applets-1.100.0.new/gweather/Makefile.am	Sun Apr 28 23:03:10 2002
@@ -31,9 +31,6 @@
 uidir = $(datadir)/gnome-2.0/ui
 ui_DATA = GNOME_GWeatherApplet.xml
 
-install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA) ;
-	
 serverdir       = $(libdir)/bonobo/servers
 server_in_files = GNOME_GWeatherApplet_Factory.server.in
 server_DATA     = $(server_in_files:.server.in=.server)
diff -Nur gnome-applets-1.100.0/mini-commander/src/Makefile.am gnome-applets-1.100.0.new/mini-commander/src/Makefile.am
--- gnome-applets-1.100.0/mini-commander/src/Makefile.am	Sun Apr 28 19:26:26 2002
+++ gnome-applets-1.100.0.new/mini-commander/src/Makefile.am	Sun Apr 28 23:02:49 2002
@@ -51,8 +51,6 @@
 uidir = $(datadir)/gnome-2.0/ui
 ui_DATA = GNOME_MiniCommanderApplet.xml
 
-install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA) ;
 
 @INTLTOOL_SERVER_RULE@
 
diff -Nur gnome-applets-1.100.0/modemlights/Makefile.am gnome-applets-1.100.0.new/modemlights/Makefile.am
--- gnome-applets-1.100.0/modemlights/Makefile.am	Sun Apr 28 19:26:31 2002
+++ gnome-applets-1.100.0.new/modemlights/Makefile.am	Sun Apr 28 23:01:39 2002
@@ -31,8 +31,3 @@
 	digits.xpm $(server_in_files)	\
 	$(schema_DATA) $(ui_DATA)
 
-install-data-local:
-	for p in $(schema_DATA); do \
-		GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$$p ; \
-	done
-
diff -Nur gnome-applets-1.100.0/multiload/Makefile.am gnome-applets-1.100.0.new/multiload/Makefile.am
--- gnome-applets-1.100.0/multiload/Makefile.am	Sun Apr 28 19:26:35 2002
+++ gnome-applets-1.100.0.new/multiload/Makefile.am	Sun Apr 28 23:02:07 2002
@@ -35,9 +35,6 @@
 schemadir 	= @GCONF_SCHEMA_FILE_DIR@
 schema_DATA	= multiload.schemas
 
-install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA) ;
-		
 EXTRA_DIST =	\
 		GNOME_MultiLoadApplet_Factory.server.in \
 		multiload.schemas \
diff -Nur gnome-applets-1.100.0/panel-menu/Makefile.am gnome-applets-1.100.0.new/panel-menu/Makefile.am
--- gnome-applets-1.100.0/panel-menu/Makefile.am	Sun Apr 28 19:26:39 2002
+++ gnome-applets-1.100.0.new/panel-menu/Makefile.am	Sun Apr 28 23:02:22 2002
@@ -63,5 +63,3 @@
 
 @INTLTOOL_SERVER_RULE@
 
-install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA) ;
diff -Nur gnome-applets-1.100.0/xmldocs.make gnome-applets-1.100.0.new/xmldocs.make
--- gnome-applets-1.100.0/xmldocs.make	Sun Apr 28 19:25:58 2002
+++ gnome-applets-1.100.0.new/xmldocs.make	Sun Apr 28 23:01:01 2002
@@ -52,10 +52,10 @@
 	done
 	touch omf_timestamp
 
-$(docname).xml: $(entities)
-        -ourdir=`pwd`;  \
-        cd $(srcdir);   \
-        cp $(entities) $$ourdir
+#$(docname).xml: $(entities)
+#        -ourdir=`pwd`;  \
+#        cd $(srcdir);   \
+#        cp $(entities) $$ourdir
 
 app-dist-hook: 
 	-$(mkinstalldirs) $(distdir)/figures
