diff -urN aaa/gnome-applets-2.15.90/configure.in gnome-applets-2.15.90/configure.in
--- aaa/gnome-applets-2.15.90/configure.in	2006-08-22 06:41:25.000000000 +0200
+++ gnome-applets-2.15.90/configure.in	2006-08-26 13:32:25.429217504 +0200
@@ -512,23 +512,32 @@
 dnl *** modemlights applet check                                            ***
 dnl ***************************************************************************
 
+dnl --disable-modemlights=(yes|no)
+AC_ARG_ENABLE(modemlights,
+	      [  --disable-modemlights=[[no/yes]]
+                          don't build modemlights applet [[default: no]]],,
+	      disable_modemlights=no)
+
 SU_TOOL=
 P_MODEMLIGHTS=
 HAVE_LIBUTIL=
+BUILD_MODEM_LIGHTS=no
 
-AC_PATH_PROG(SU_TOOL, su)
-AC_CHECK_LIB(util, forkpty, HAVE_LIBUTIL=true, HAVE_LIBUTIL=false)
-AM_PATH_SYSTEM_TOOLS_BACKENDS($SYSTEM_TOOLS_BACKENDS_REQUIRED, P_MODEMLIGHTS=modemlights)
-
-if test "x$HAVE_LIBUTIL" = "xtrue" -a "x$P_MODEMLIGHTS" = "xmodemlights" -a "x$HAVE_GNOME_SETTINGS_DAEMON" = "xyes"; then
-  MODEMLIGHTS_LIBS=" -lutil"
-  AC_SUBST(MODEMLIGHTS_LIBS)
-  AC_DEFINE_UNQUOTED(STB_SCRIPTS_DIR, "$STB_SCRIPTS_DIR", [System tools backends])
-  AC_DEFINE_UNQUOTED(SU_PATH, "${SU_TOOL}", [su executable path])
-  BUILD_MODEM_LIGHTS=yes
-else
-  AC_MSG_WARN([*** modemlights applet will not be built ***])
-  BUILD_MODEM_LIGHTS=no
+if test x$disable_modemlights = xno; then
+  AC_PATH_PROG(SU_TOOL, su)
+  AC_CHECK_LIB(util, forkpty, HAVE_LIBUTIL=true, HAVE_LIBUTIL=false)
+  AM_PATH_SYSTEM_TOOLS_BACKENDS($SYSTEM_TOOLS_BACKENDS_REQUIRED, P_MODEMLIGHTS=modemlights)
+
+  if test "x$HAVE_LIBUTIL" = "xtrue" -a "x$P_MODEMLIGHTS" = "xmodemlights" -a "x$HAVE_GNOME_SETTINGS_DAEMON" = "xyes"; then
+    MODEMLIGHTS_LIBS=" -lutil"
+    AC_SUBST(MODEMLIGHTS_LIBS)
+    AC_DEFINE_UNQUOTED(STB_SCRIPTS_DIR, "$STB_SCRIPTS_DIR", [System tools backends])
+    AC_DEFINE_UNQUOTED(SU_PATH, "${SU_TOOL}", [su executable path])
+    BUILD_MODEM_LIGHTS=yes
+  else
+    AC_MSG_WARN([*** modemlights applet will not be built ***])
+    BUILD_MODEM_LIGHTS=no
+  fi
 fi
 
 AM_CONDITIONAL(APPLET_MODEMLIGHTS, test "x$BUILD_MODEM_LIGHTS" = "xyes")
