--- gnome-applets-1.4.0.5/sound-monitor/themes/Makefile.am~	Sat Jun 15 22:12:39 2002
+++ gnome-applets-1.4.0.5/sound-monitor/themes/Makefile.am	Sat Jun 15 22:12:39 2002
@@ -1,6 +1,5 @@
 radar_bluedir = $(datadir)/sound-monitor/radar-blue
-radar_blue_DATA =
-radar_blue_stuff = \
+radar_blue_DATA =			\
 	radar-blue/back.png		\
 	radar-blue/scope.png		\
 	radar-blue/back_tiny_h.png	\
@@ -10,8 +9,7 @@
 	radar-blue/themedata
 
 radar_greendir = $(datadir)/sound-monitor/radar-green
-radar_green_DATA =
-radar_green_stuff = \
+radar_green_DATA =			\
 	radar-green/back.png		\
 	radar-green/scope.png		\
 	radar-green/back_tiny_h.png	\
@@ -21,8 +19,7 @@
 	radar-green/themedata
 
 bluedir = $(datadir)/sound-monitor/blue
-blue_DATA =
-blue_stuff = \
+blue_DATA =			\
 	blue/back_h.png		\
 	blue/back_v.png		\
 	blue/back_tiny.png	\
@@ -32,16 +29,14 @@
 	blue/themedata
 
 metersdir = $(datadir)/sound-monitor/meters
-meters_DATA =
-meters_stuff = \
+meters_DATA =			\
 	meters/back.png		\
 	meters/meter.png	\
 	meters/meter.xcf.gz	\
 	meters/status.png	\
 	meters/themedata
 originaldir = $(datadir)/sound-monitor/original
-original_DATA =
-original_stuff = \
+original_DATA =			\
 	original/back_h.png	\
 	original/back_v.png	\
 	original/meter_h.png	\
@@ -49,8 +44,7 @@
 	original/themedata
 
 scope_ldir = $(datadir)/sound-monitor/scope-larger
-scope_l_DATA =
-scope_l_stuff = \
+scope_l_DATA =				\
 	scope-larger/back_h.png		\
 	scope-larger/back_v.png		\
 	scope-larger/scope_h.png	\
@@ -62,8 +56,7 @@
 	scope-larger/themedata
 
 greydir = $(datadir)/sound-monitor/grey
-grey_DATA =
-grey_stuff = \
+grey_DATA =			\
 	grey/background_h.png	\
 	grey/background_v.png	\
 	grey/meter_h.png	\
@@ -73,8 +66,7 @@
 	grey/themedata
 
 meters2dir = $(datadir)/sound-monitor/meters2
-meters2_DATA =
-meters2_stuff = \
+meters2_DATA =			\
 	meters2/back.png	\
 	meters2/back_tiny_v.png	\
 	meters2/meter.png	\
@@ -83,8 +75,7 @@
 	meters2/themedata
 
 scopedir = $(datadir)/sound-monitor/scope
-scope_DATA =
-scope_stuff = \
+scope_DATA =			\
 	scope/back.png		\
 	scope/scope.png		\
 	scope/back_tiny_h.png	\
@@ -94,8 +85,7 @@
 	scope/themedata
 
 scope_rdir = $(datadir)/sound-monitor/scope-round
-scope_r_DATA =
-scope_r_stuff = \
+scope_r_DATA =				\
 	scope-round/back.png		\
 	scope-round/scope.png		\
 	scope-round/back_tiny.png	\
@@ -103,8 +93,7 @@
 	scope-round/themedata
 
 scope_cleardir = $(datadir)/sound-monitor/scope-clear
-scope_clear_DATA =
-scope_clear_stuff = \
+scope_clear_DATA =			\
 	scope-clear/back.png		\
 	scope-clear/back_small_h.png	\
 	scope-clear/back_small_v.png	\
@@ -125,18 +114,16 @@
 #	spectra-clear/analyzer.png	\
 #	spectra-clear/themedata
 
-zsound_monitordir = $(datadir)/sound-monitor
-zsound_monitor_DATA = \
-	$(radar_blue_stuff)		\
-	$(radar_green_stuff)		\
-	$(blue_stuff)		\
-	$(meters_stuff)		\
-	$(original_stuff)	\
-	$(scope_l_stuff)	\
-	$(grey_stuff)		\
-	$(meters2_stuff)	\
-	$(scope_stuff)		\
-	$(scope_r_stuff)	\
-	$(scope_clear_stuff)
 
-EXTRA_DIST = $(zsound_monitor_DATA)
+EXTRA_DIST = 			\
+	$(radar_blue_DATA)	\
+	$(radar_green_DATA)	\
+	$(blue_DATA)		\
+	$(meters_DATA)		\
+	$(original_DATA)	\
+	$(scope_l_DATA)		\
+	$(grey_DATA)		\
+	$(meters2_DATA)		\
+	$(scope_DATA)		\
+	$(scope_r_DATA)		\
+	$(scope_clear_DATA)
