--- gnome-applets-1.4.0.5/asclock/themes/Makefile.am~	Sat Jun 15 23:26:01 2002
+++ gnome-applets-1.4.0.5/asclock/themes/Makefile.am	Sat Jun 15 23:26:01 2002
@@ -1,8 +1,7 @@
 #SUBDIRS = Orb classic shaped beats Stone Newstone Freeamp
 
 Orbdir = $(datadir)/asclock/Orb
-Orb_DATA = 
-Orb_stuff = \
+Orb_DATA =		\
 	Orb/config	\
 	Orb/clock.xpm	\
 	Orb/led.xpm	\
@@ -15,11 +14,10 @@
 	Orb/weekday.xpm
 
 classicdir = $(datadir)/asclock/classic
-classic_DATA =
-classic_stuff = \
-	classic/config	\
+classic_DATA =			\
+	classic/config		\
 	classic/clock.xpm	\
-	classic/led.xpm	\
+	classic/led.xpm		\
 	classic/date.xpm	\
 	classic/month.xpm	\
 	classic/hour.xpm	\
@@ -29,51 +27,47 @@
 	classic/weekday.xpm
 
 shapeddir = $(datadir)/asclock/shaped
-shaped_DATA =
-shaped_stuff = \
-	shaped/config	\
+shaped_DATA =			\
+	shaped/config		\
 	shaped/clock.xpm	\
-	shaped/led.xpm	\
-	shaped/date.xpm	\
+	shaped/led.xpm		\
+	shaped/date.xpm		\
 	shaped/month.xpm	\
-	shaped/hour.xpm	\
+	shaped/hour.xpm		\
 	shaped/minute.xpm	\
 	shaped/second.xpm	\
 	shaped/beats.xpm	\
 	shaped/weekday.xpm
 
 beatsdir = $(datadir)/asclock/beats
-beats_DATA =
-beats_stuff = \
-	beats/config	\
-	beats/clock.xpm	\
-	beats/led.xpm	\
-	beats/date.xpm	\
-	beats/month.xpm	\
-	beats/hour.xpm	\
+beats_DATA =			\
+	beats/config		\
+	beats/clock.xpm		\
+	beats/led.xpm		\
+	beats/date.xpm		\
+	beats/month.xpm		\
+	beats/hour.xpm		\
 	beats/minute.xpm	\
 	beats/second.xpm	\
-	beats/beats.xpm	\
+	beats/beats.xpm		\
 	beats/weekday.xpm
 
 Stonedir = $(datadir)/asclock/Stone
-Stone_DATA =
-Stone_stuff = \
-	Stone/config	\
-	Stone/clock.xpm	\
-	Stone/led.xpm	\
-	Stone/date.xpm	\
-	Stone/month.xpm	\
-	Stone/hour.xpm	\
+Stone_DATA =			\
+	Stone/config		\
+	Stone/clock.xpm		\
+	Stone/led.xpm		\
+	Stone/date.xpm		\
+	Stone/month.xpm		\
+	Stone/hour.xpm		\
 	Stone/minute.xpm	\
 	Stone/second.xpm	\
-	Stone/beats.xpm	\
+	Stone/beats.xpm		\
 	Stone/weekday.xpm
 
 Newstonedir = $(datadir)/asclock/Newstone
-Newstone_DATA =
-Newstone_stuff = \
-	Newstone/config	\
+Newstone_DATA =			\
+	Newstone/config		\
 	Newstone/clock.xpm	\
 	Newstone/led.xpm	\
 	Newstone/date.xpm	\
@@ -85,11 +79,10 @@
 	Newstone/weekday.xpm
 
 Freeampdir = $(datadir)/asclock/Freeamp
-Freeamp_DATA =
-Freeamp_stuff = \
-	Freeamp/config	\
+Freeamp_DATA =			\
+	Freeamp/config		\
 	Freeamp/clock.xpm	\
-	Freeamp/led.xpm	\
+	Freeamp/led.xpm		\
 	Freeamp/date.xpm	\
 	Freeamp/month.xpm	\
 	Freeamp/hour.xpm	\
@@ -98,14 +91,11 @@
 	Freeamp/beats.xpm	\
 	Freeamp/weekday.xpm
 
-zasclockdir = $(datadir)/asclock
-zasclock_DATA = \
-	$(Orb_stuff)		\
-	$(classic_stuff)	\
-	$(shaped_stuff)		\
-	$(beats_stuff)		\
-	$(Stone_stuff)		\
-	$(Newstone_stuff)	\
-	$(Freeamp_stuff)
-
-EXTRA_DIST = $(zasclock_DATA)
\ No newline at end of file
+EXTRA_DIST = 			\
+	$(Orb_DATA)		\
+	$(classic_DATA)		\
+	$(shaped_DATA)		\
+	$(beats_DATA)		\
+	$(Stone_DATA)		\
+	$(Newstone_DATA)	\
+	$(Freeamp_DATA)
