--- gnome-applets-2.21.4/gweather/gweather-xml.c~	2008-01-15 00:05:22.000000000 +0100
+++ gnome-applets-2.21.4/gweather/gweather-xml.c	2008-01-16 13:49:03.203654461 +0100
@@ -357,7 +357,7 @@
   int ret = -1;
 
   /* Open the xml file containing the different locations */
-  xml = xmlNewTextReaderFilename (GWEATHER_XML_LOCATION "Locations.xml");
+  xml = xmlNewTextReaderFilename (GWEATHER_XML_LOCATION);
   if( xml == NULL )
     goto error_out;
 
