diff -urN gnome-applets-2.24.0.1/configure.in gnome-applets-2.24.0.1.new/configure.in
--- gnome-applets-2.24.0.1/configure.in	2008-09-24 18:45:20.000000000 +0200
+++ gnome-applets-2.24.0.1.new/configure.in	2008-10-06 20:57:18.000000000 +0200
@@ -555,13 +555,12 @@
 dnl ***************************************************************************
 
 SU_TOOL=
-P_MODEMLIGHTS=
 HAVE_LIBUTIL=
 
 AC_PATH_PROG(SU_TOOL, su)
 AC_CHECK_LIB(util, forkpty, HAVE_LIBUTIL=true, HAVE_LIBUTIL=false)
 
-if test "x$HAVE_LIBUTIL" = "xtrue" -a "x$P_MODEMLIGHTS" = "xmodemlights" -a "x$HAVE_GNOME_SETTINGS_DAEMON" = "xyes"; then
+if test "x$HAVE_LIBUTIL" = "xtrue" -a "x$HAVE_GNOME_SETTINGS_DAEMON" = "xyes"; then
   MODEMLIGHTS_LIBS=" -lutil"
   AC_SUBST(MODEMLIGHTS_LIBS)
   AC_DEFINE_UNQUOTED(STB_SCRIPTS_DIR, "$STB_SCRIPTS_DIR", [System tools backends])
