--- gnome-applets-1.4.0.5/geyes/Makefile.am~	Sat May 18 23:26:15 2002
+++ gnome-applets-1.4.0.5/geyes/Makefile.am	Sat May 18 23:26:15 2002
@@ -16,11 +16,6 @@
 	$(GNOME_LIBDIR) \
 	$(GNOME_APPLETS_LIBS) $(INTLLIBS)
 
-EXTRA_DIST = geyes_applet.desktop.in \
-	     geyes_applet.gnorba \
-	     geyes.h \
-	     gnome-eyes.png
-
 sysdir = $(datadir)/applets/Amusements
 sys_predata = geyes_applet.desktop.in
 sys_DATA = $(sys_predata:.desktop.in=.desktop)
@@ -36,9 +31,12 @@
 
 man_MANS = geyes_applet.1
 
-EXTRA_DIST = README.themes $(pixmap_DATA)	\
-	$(gnorba_DATA) $(sys_DATA)		\
-	$(sys_predata)				\
-	$(man_MANS)
+EXTRA_DIST = geyes_applet.desktop.in 		\
+	     geyes_applet.gnorba 		\
+	     geyes.h 				\
+	     gnome-eyes.png			\
+	     README.themes $(pixmap_DATA)	\
+	     $(gnorba_DATA) $(sys_DATA)		\
+	     $(sys_predata) $(man_MANS)
 
 
--- gnome-applets-1.4.0.5/applet-docs.make~	Sat May 18 23:44:37 2002
+++ gnome-applets-1.4.0.5/applet-docs.make	Sat May 18 23:44:37 2002
@@ -36,7 +36,7 @@
 	-cp $(applet)_applet/index.html .
 
 $(applet)_applet.sgml: $(sgml_ents)
-	-ourdir=`pwd`;	\
+	@ourdir=`pwd`;	\
 	cd $(srcdir);	\
 	cp $(sgml_ents) $$ourdir
 
--- gnome-applets-1.4.0.5/sgmldocs.make~	Sun May 19 00:13:19 2002
+++ gnome-applets-1.4.0.5/sgmldocs.make	Sun May 19 00:13:19 2002
@@ -58,7 +58,7 @@
 	-cp $(docname)/index.html .
 
 $(docname).sgml: $(sgml_ents)
-        -ourdir=`pwd`;  \
+        @ourdir=`pwd`;  \
         cd $(srcdir);   \
         cp $(sgml_ents) $$ourdir
 
--- gnome-applets-1.4.0.5/cpumemusage/docs/ru/Makefile.am~	Sun May 19 00:23:26 2002
+++ gnome-applets-1.4.0.5/cpumemusage/docs/ru/Makefile.am	Sun May 19 00:23:26 2002
@@ -1,5 +1,8 @@
+figs = \
+	figures/cpumemusage-applet.png
 applet = cpumemusage
 lang = ru
+omffile = cpumemusage_applet-ru.omf
 sgml_ents = $(applet).sgml
 include $(top_srcdir)/applet-docs.make
 dist-hook: applet-dist-hook
