diff -aurN gnome-applets-2.15.3.orig/libgswitchit/keyboard-config-registry.c gnome-applets-2.15.3/libgswitchit/keyboard-config-registry.c
--- gnome-applets-2.15.3.orig/libgswitchit/keyboard-config-registry.c	2006-08-07 15:52:13.000000000 +0200
+++ gnome-applets-2.15.3/libgswitchit/keyboard-config-registry.c	2006-08-09 21:48:23.178132370 +0200
@@ -146,7 +146,7 @@
 						  DBUS_PATH_DBUS,
 						  DBUS_INTERFACE_DBUS);
 
-	if (!org_freedesktop_DBus_request_name
+	if (!dbus_bus_request_name
 	    (driver_proxy, "org.gnome.KeyboardConfigRegistry", 0,
 	     &request_ret, &error)) {
 		g_warning ("Unable to register service: %s",
diff -aurN gnome-applets-2.15.3.orig/libgswitchit/keyboard-config-registry.h gnome-applets-2.15.3/libgswitchit/keyboard-config-registry.h
--- gnome-applets-2.15.3.orig/libgswitchit/keyboard-config-registry.h	2006-07-06 21:42:56.000000000 +0200
+++ gnome-applets-2.15.3/libgswitchit/keyboard-config-registry.h	2006-08-09 21:47:09.971261506 +0200
@@ -1,7 +1,7 @@
 #ifndef __KEYBOARD_CONFIG_REGISTRY_H__
 #define __KEYBOARD_CONFIG_REGISTRY_H__
 
-#include <dbus/dbus-glib-bindings.h>
+#include <dbus/dbus-glib.h>
 #include <libxklavier/xklavier.h>
 
 typedef struct KeyboardConfigRegistry KeyboardConfigRegistry;
