diff -urN gnome-applets-1.4.0.1.org/configure.in gnome-applets-1.4.0.1/configure.in
--- gnome-applets-1.4.0.1.org/configure.in	Sat Apr  7 20:13:29 2001
+++ gnome-applets-1.4.0.1/configure.in	Sat Apr  7 20:23:47 2001
@@ -206,7 +206,7 @@
 AC_SUBST(CANVAS_PIXBUF_LIBS)
 
 ALL_LINGUAS="az ca cs da de el en_GB es et eu fi fr ga gl hu it ja ko lt nl no pl pt pt_BR ro ru sk sl sv tr uk wa zh_CN.GB2312 zh_TW.Big5"
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 # this is the directory where the *.{mo,gmo} files are installed
 gnomelocaledir='${prefix}/${DATADIRNAME}/locale'
 AC_SUBST(gnomelocaledir)
