diff -aurN gnome-applets-2.12.1.orig/mixer/applet.c gnome-applets-2.12.1/mixer/applet.c
--- gnome-applets-2.12.1.orig/mixer/applet.c	2005-10-04 03:02:17.000000000 +0200
+++ gnome-applets-2.12.1/mixer/applet.c	2005-10-08 11:14:49.000000000 +0200
@@ -716,7 +716,7 @@
       case GDK_SCROLL_UP:
       case GDK_SCROLL_DOWN: {
         GtkAdjustment *adj = gtk_range_get_adjustment (applet->dock->scale);
-        gfloat volume = adj->value;
+        gint volume = adj->value;
 
         if (event->direction == GDK_SCROLL_UP) {
           volume += adj->step_increment;
@@ -942,7 +942,7 @@
 void
 gnome_volume_applet_adjust_volume (GstMixer *mixer,
 				   GstMixerTrack *track,
-				   float volume)
+				   int volume)
 {
   int range = track->max_volume - track->min_volume;
   float scale = ((float) range) / 100;
@@ -953,7 +953,7 @@
 
   volumes = g_new (gint, track->num_channels);
   for (n = 0; n < track->num_channels; n++)
-    volumes[n] = lrintf (volume);
+    volumes[n] = volume;
   gst_mixer_set_volume (mixer, track, volumes);
   g_free (volumes);
 }
@@ -989,8 +989,7 @@
 	   gpointer data)
 {
   GnomeVolumeApplet *applet = data;
-  gint *volumes, n;
-  gfloat v;
+  gint *volumes, n, v;
   GList *iter;
 
   if (applet->lock)
diff -aurN gnome-applets-2.12.1.orig/mixer/applet.h gnome-applets-2.12.1/mixer/applet.h
--- gnome-applets-2.12.1.orig/mixer/applet.h	2005-10-04 03:02:17.000000000 +0200
+++ gnome-applets-2.12.1/mixer/applet.h	2005-10-08 11:14:49.000000000 +0200
@@ -92,7 +92,7 @@
 
 void     gnome_volume_applet_adjust_volume (GstMixer      *mixer,
 					    GstMixerTrack *track,
-					    float          volume);
+					    int            volume);
 GType    gnome_volume_applet_get_type (void);
 gboolean gnome_volume_applet_setup    (GnomeVolumeApplet *applet,
 				       GList             *elements);
diff -aurN gnome-applets-2.12.1.orig/mixer/dock.c gnome-applets-2.12.1/mixer/dock.c
--- gnome-applets-2.12.1.orig/mixer/dock.c	2005-10-04 03:02:17.000000000 +0200
+++ gnome-applets-2.12.1/mixer/dock.c	2005-10-08 11:14:49.000000000 +0200
@@ -196,7 +196,7 @@
 {
   GnomeVolumeAppletDock *dock = data;
   GtkAdjustment *adj;
-  gfloat volume;
+  gint volume;
   gboolean res = TRUE;
 
   if (!dock->timeout)
