--- gnome-applets-1.4.0.5/asclock/themes/Makefile.am~	Sat Jun 15 23:26:01 2002
+++ gnome-applets-1.4.0.5/asclock/themes/Makefile.am	Sat Jun 15 23:26:01 2002
@@ -1,8 +1,7 @@
 #SUBDIRS = Orb classic shaped beats Stone Newstone Freeamp
 
 Orbdir = $(datadir)/asclock/Orb
-Orb_DATA = 
-Orb_stuff = \
+Orb_DATA =		\
 	Orb/config	\
 	Orb/clock.xpm	\
 	Orb/led.xpm	\
@@ -15,11 +14,10 @@
 	Orb/weekday.xpm
 
 classicdir = $(datadir)/asclock/classic
-classic_DATA =
-classic_stuff = \
-	classic/config	\
+classic_DATA =			\
+	classic/config		\
 	classic/clock.xpm	\
-	classic/led.xpm	\
+	classic/led.xpm		\
 	classic/date.xpm	\
 	classic/month.xpm	\
 	classic/hour.xpm	\
@@ -29,51 +27,47 @@
 	classic/weekday.xpm
 
 shapeddir = $(datadir)/asclock/shaped
-shaped_DATA =
-shaped_stuff = \
-	shaped/config	\
+shaped_DATA =			\
+	shaped/config		\
 	shaped/clock.xpm	\
-	shaped/led.xpm	\
-	shaped/date.xpm	\
+	shaped/led.xpm		\
+	shaped/date.xpm		\
 	shaped/month.xpm	\
-	shaped/hour.xpm	\
+	shaped/hour.xpm		\
 	shaped/minute.xpm	\
 	shaped/second.xpm	\
 	shaped/beats.xpm	\
 	shaped/weekday.xpm
 
 beatsdir = $(datadir)/asclock/beats
-beats_DATA =
-beats_stuff = \
-	beats/config	\
-	beats/clock.xpm	\
-	beats/led.xpm	\
-	beats/date.xpm	\
-	beats/month.xpm	\
-	beats/hour.xpm	\
+beats_DATA =			\
+	beats/config		\
+	beats/clock.xpm		\
+	beats/led.xpm		\
+	beats/date.xpm		\
+	beats/month.xpm		\
+	beats/hour.xpm		\
 	beats/minute.xpm	\
 	beats/second.xpm	\
-	beats/beats.xpm	\
+	beats/beats.xpm		\
 	beats/weekday.xpm
 
 Stonedir = $(datadir)/asclock/Stone
-Stone_DATA =
-Stone_stuff = \
-	Stone/config	\
-	Stone/clock.xpm	\
-	Stone/led.xpm	\
-	Stone/date.xpm	\
-	Stone/month.xpm	\
-	Stone/hour.xpm	\
+Stone_DATA =			\
+	Stone/config		\
+	Stone/clock.xpm		\
+	Stone/led.xpm		\
+	Stone/date.xpm		\
+	Stone/month.xpm		\
+	Stone/hour.xpm		\
 	Stone/minute.xpm	\
 	Stone/second.xpm	\
-	Stone/beats.xpm	\
+	Stone/beats.xpm		\
 	Stone/weekday.xpm
 
 Newstonedir = $(datadir)/asclock/Newstone
-Newstone_DATA =
-Newstone_stuff = \
-	Newstone/config	\
+Newstone_DATA =			\
+	Newstone/config		\
 	Newstone/clock.xpm	\
 	Newstone/led.xpm	\
 	Newstone/date.xpm	\
@@ -85,11 +79,10 @@
 	Newstone/weekday.xpm
 
 Freeampdir = $(datadir)/asclock/Freeamp
-Freeamp_DATA =
-Freeamp_stuff = \
-	Freeamp/config	\
+Freeamp_DATA =			\
+	Freeamp/config		\
 	Freeamp/clock.xpm	\
-	Freeamp/led.xpm	\
+	Freeamp/led.xpm		\
 	Freeamp/date.xpm	\
 	Freeamp/month.xpm	\
 	Freeamp/hour.xpm	\
@@ -98,14 +91,11 @@
 	Freeamp/beats.xpm	\
 	Freeamp/weekday.xpm
 
-zasclockdir = $(datadir)/asclock
-zasclock_DATA = \
-	$(Orb_stuff)		\
-	$(classic_stuff)	\
-	$(shaped_stuff)		\
-	$(beats_stuff)		\
-	$(Stone_stuff)		\
-	$(Newstone_stuff)	\
-	$(Freeamp_stuff)
-
-EXTRA_DIST = $(zasclock_DATA)
\ No newline at end of file
+EXTRA_DIST = 			\
+	$(Orb_DATA)		\
+	$(classic_DATA)		\
+	$(shaped_DATA)		\
+	$(beats_DATA)		\
+	$(Stone_DATA)		\
+	$(Newstone_DATA)	\
+	$(Freeamp_DATA)
--- gnome-applets-1.4.0.5/sound-monitor/themes/Makefile.am~	Sat Jun 15 22:12:39 2002
+++ gnome-applets-1.4.0.5/sound-monitor/themes/Makefile.am	Sat Jun 15 22:12:39 2002
@@ -1,6 +1,5 @@
 radar_bluedir = $(datadir)/sound-monitor/radar-blue
-radar_blue_DATA =
-radar_blue_stuff = \
+radar_blue_DATA =			\
 	radar-blue/back.png		\
 	radar-blue/scope.png		\
 	radar-blue/back_tiny_h.png	\
@@ -10,8 +9,7 @@
 	radar-blue/themedata
 
 radar_greendir = $(datadir)/sound-monitor/radar-green
-radar_green_DATA =
-radar_green_stuff = \
+radar_green_DATA =			\
 	radar-green/back.png		\
 	radar-green/scope.png		\
 	radar-green/back_tiny_h.png	\
@@ -21,8 +19,7 @@
 	radar-green/themedata
 
 bluedir = $(datadir)/sound-monitor/blue
-blue_DATA =
-blue_stuff = \
+blue_DATA =			\
 	blue/back_h.png		\
 	blue/back_v.png		\
 	blue/back_tiny.png	\
@@ -32,16 +29,14 @@
 	blue/themedata
 
 metersdir = $(datadir)/sound-monitor/meters
-meters_DATA =
-meters_stuff = \
+meters_DATA =			\
 	meters/back.png		\
 	meters/meter.png	\
 	meters/meter.xcf.gz	\
 	meters/status.png	\
 	meters/themedata
 originaldir = $(datadir)/sound-monitor/original
-original_DATA =
-original_stuff = \
+original_DATA =			\
 	original/back_h.png	\
 	original/back_v.png	\
 	original/meter_h.png	\
@@ -49,8 +44,7 @@
 	original/themedata
 
 scope_ldir = $(datadir)/sound-monitor/scope-larger
-scope_l_DATA =
-scope_l_stuff = \
+scope_l_DATA =				\
 	scope-larger/back_h.png		\
 	scope-larger/back_v.png		\
 	scope-larger/scope_h.png	\
@@ -62,8 +56,7 @@
 	scope-larger/themedata
 
 greydir = $(datadir)/sound-monitor/grey
-grey_DATA =
-grey_stuff = \
+grey_DATA =			\
 	grey/background_h.png	\
 	grey/background_v.png	\
 	grey/meter_h.png	\
@@ -73,8 +66,7 @@
 	grey/themedata
 
 meters2dir = $(datadir)/sound-monitor/meters2
-meters2_DATA =
-meters2_stuff = \
+meters2_DATA =			\
 	meters2/back.png	\
 	meters2/back_tiny_v.png	\
 	meters2/meter.png	\
@@ -83,8 +75,7 @@
 	meters2/themedata
 
 scopedir = $(datadir)/sound-monitor/scope
-scope_DATA =
-scope_stuff = \
+scope_DATA =			\
 	scope/back.png		\
 	scope/scope.png		\
 	scope/back_tiny_h.png	\
@@ -94,8 +85,7 @@
 	scope/themedata
 
 scope_rdir = $(datadir)/sound-monitor/scope-round
-scope_r_DATA =
-scope_r_stuff = \
+scope_r_DATA =				\
 	scope-round/back.png		\
 	scope-round/scope.png		\
 	scope-round/back_tiny.png	\
@@ -103,8 +93,7 @@
 	scope-round/themedata
 
 scope_cleardir = $(datadir)/sound-monitor/scope-clear
-scope_clear_DATA =
-scope_clear_stuff = \
+scope_clear_DATA =			\
 	scope-clear/back.png		\
 	scope-clear/back_small_h.png	\
 	scope-clear/back_small_v.png	\
@@ -125,18 +114,16 @@
 #	spectra-clear/analyzer.png	\
 #	spectra-clear/themedata
 
-zsound_monitordir = $(datadir)/sound-monitor
-zsound_monitor_DATA = \
-	$(radar_blue_stuff)		\
-	$(radar_green_stuff)		\
-	$(blue_stuff)		\
-	$(meters_stuff)		\
-	$(original_stuff)	\
-	$(scope_l_stuff)	\
-	$(grey_stuff)		\
-	$(meters2_stuff)	\
-	$(scope_stuff)		\
-	$(scope_r_stuff)	\
-	$(scope_clear_stuff)
 
-EXTRA_DIST = $(zsound_monitor_DATA)
+EXTRA_DIST = 			\
+	$(radar_blue_DATA)	\
+	$(radar_green_DATA)	\
+	$(blue_DATA)		\
+	$(meters_DATA)		\
+	$(original_DATA)	\
+	$(scope_l_DATA)		\
+	$(grey_DATA)		\
+	$(meters2_DATA)		\
+	$(scope_DATA)		\
+	$(scope_r_DATA)		\
+	$(scope_clear_DATA)
--- gnome-applets-1.4.0.5/clockmail/themes/Makefile.am.orig	Fri Oct 26 11:10:38 2001
+++ gnome-applets-1.4.0.5/clockmail/themes/Makefile.am	Fri Sep  6 12:04:57 2002
@@ -1,6 +1,5 @@
 analogdir = $(datadir)/clockmail/analog-clock-blue
-analog_DATA = 
-analog_stuff = \
+analog_DATA = \
 	analog-clock-blue/back.png	\
 	analog-clock-blue/btn.png	\
 	analog-clock-blue/day_en.png	\
@@ -12,8 +11,7 @@
 	analog-clock-blue/clockmaildata
 
 calendardir = $(datadir)/clockmail/calendar
-calendar_DATA =
-calendar_stuff = \
+calendar_DATA = \
 	calendar/backgrnd.png		\
 	calendar/backgrnd_v.png		\
 	calendar/backgrnd_tiny.png	\
@@ -25,16 +23,14 @@
 	calendar/clockmaildata
 
 classicdir = $(datadir)/clockmail/classic
-classic_DATA =
-classic_stuff = \
+classic_DATA = \
 	classic/backgrnd.png		\
 	classic/digmed.png		\
 	classic/mailpics.png		\
 	classic/clockmaildata
 
 clock_analogdir = $(datadir)/clockmail/clock-analog
-clock_analog_DATA =
-clock_analog_stuff = \
+clock_analog_DATA = \
 	clock-analog/back.png		\
 	clock-analog/hour_hand.png	\
 	clock-analog/minute_hand.png	\
@@ -50,8 +46,7 @@
 	clock-analog/clockmaildata
 
 clock_odometerdir = $(datadir)/clockmail/clock-odometer
-clock_odometer_DATA =
-clock_odometer_stuff = \
+clock_odometer_DATA = \
 	clock-odometer/back_h.png	\
 	clock-odometer/back_v.png	\
 	clock-odometer/back_tiny_h.png	\
@@ -61,8 +56,7 @@
 	clock-odometer/clockmaildata
 
 Gdir = $(datadir)/clockmail/clockmail-G
-G_DATA =
-G_stuff = \
+G_DATA = \
 	clockmail-G/back.png		\
 	clockmail-G/button.png		\
 	clockmail-G/days_en.png		\
@@ -74,8 +68,7 @@
 	clockmail-G/clockmaildata
 
 lcddir = $(datadir)/clockmail/lcd
-lcd_DATA = 
-lcd_stuff = \
+lcd_DATA = \
 	lcd/background.png		\
 	lcd/digits.png			\
 	lcd/graph.png			\
@@ -83,39 +76,34 @@
 	lcd/clockmaildata
 
 mailflapdir = $(datadir)/clockmail/mail-flap
-mailflap_DATA = 
-mailflap_stuff = \
+mailflap_DATA = \
 	mail-flap/background.png	\
 	mail-flap/mail.png		\
 	mail-flap/clockmaildata
 
 fuelgaugedir = $(datadir)/clockmail/mail-fuel-gauge
-fuelgauge_DATA =
-fuelgauge_stuff = \
+fuelgauge_DATA = \
 	mail-fuel-gauge/background.png	\
 	mail-fuel-gauge/light.png	\
 	mail-fuel-gauge/pointer.png	\
 	mail-fuel-gauge/clockmaildata
 
 counterdir = $(datadir)/clockmail/mailcounter
-counter_DATA =
-counter_stuff = \
+counter_DATA = \
 	mailcounter/backgrnd.png	\
 	mailcounter/digmed.png		\
 	mailcounter/mailpics.png	\
 	mailcounter/clockmaildata
 
 odometerdir = $(datadir)/clockmail/odometer
-odometer_DATA =
-odometer_stuff = \
+odometer_DATA = \
 	odometer/backgrnd.png		\
 	odometer/diglrg.png		\
 	odometer/mail.png		\
 	odometer/clockmaildata
 
 radar_bluedir = $(datadir)/clockmail/radar-blue
-radar_blue_DATA =
-radar_blue_stuff = \
+radar_blue_DATA = \
 	radar-blue/clockmaildata	\
 	radar-blue/hour_hand.png	\
 	radar-blue/minute_hand.png	\
@@ -123,8 +111,7 @@
 	radar-blue/back.png
 
 radar_greendir = $(datadir)/clockmail/radar-green
-radar_green_DATA =
-radar_green_stuff = \
+radar_green_DATA = \
 	radar-green/clockmaildata	\
 	radar-green/hour_hand.png	\
 	radar-green/minute_hand.png	\
@@ -132,8 +119,7 @@
 	radar-green/back.png
 
 nice_clockdir = $(datadir)/clockmail/nice_clock
-nice_clock_DATA =
-nice_clock_stuff = \
+nice_clock_DATA = \
 	nice_clock/clockmaildata	\
 	nice_clock/hour_hand.png	\
 	nice_clock/minute_hand.png	\
@@ -145,8 +131,7 @@
         nice_clock/back_tiny.png
 
 nice_mailcountdir = $(datadir)/clockmail/nice_mailcount
-nice_mailcount_DATA =
-nice_mailcount_stuff = \
+nice_mailcount_DATA = \
         nice_mailcount/back.png		\
 	nice_mailcount/back_h.png	\
         nice_mailcount/digit.png	\
@@ -156,8 +141,7 @@
         nice_mailcount/clockmaildata
 
 clock_digital_blackdir = $(datadir)/clockmail/clock-digital-black
-clock_digital_black_DATA =
-clock_digital_black_stuff = \
+clock_digital_black_DATA = \
 	clock-digital-black/bback.png		\
 	clock-digital-black/bdigits.png		\
 	clock-digital-black/bback_sm.png	\
@@ -165,8 +149,7 @@
 	clock-digital-black/clockmaildata
 
 clock_digital_reddir = $(datadir)/clockmail/clock-digital-red
-clock_digital_red_DATA =
-clock_digital_red_stuff = \
+clock_digital_red_DATA = \
 	clock-digital-red/back.png	\
 	clock-digital-red/digits.png	\
 	clock-digital-red/back_sm.png	\
@@ -174,8 +157,7 @@
 	clock-digital-red/clockmaildata
 
 green_analog_digitaldir = $(datadir)/clockmail/green-analog-digital
-green_analog_digital_DATA =
-green_analog_digital_stuff = \
+green_analog_digital_DATA = \
 	green-analog-digital/back.png		\
 	green-analog-digital/btn.png		\
 	green-analog-digital/day_en.png		\
@@ -188,24 +170,23 @@
 	green-analog-digital/clockmaildata
 	
 zclockmaildir = $(datadir)/clockmail
-zclockmail_DATA = \
-	$(analog_stuff)		\
-	$(calendar_stuff)	\
-	$(classic_stuff)	\
-	$(clock_analog_stuff)	\
-	$(clock_odometer_stuff)	\
-	$(G_stuff)		\
-	$(lcd_stuff)		\
-	$(mailflap_stuff)	\
-	$(fuelgauge_stuff)	\
-	$(counter_stuff)	\
-	$(odometer_stuff)	\
-	$(radar_blue_stuff)	\
-	$(radar_green_stuff)	\
-	$(nice_clock_stuff)	\
-	$(nice_mailcount_stuff)	\
-	$(clock_digital_black_stuff)	\
-	$(clock_digital_red_stuff)	\
-	$(green_analog_digital_stuff)
-	
-EXTRA_DIST = $(zclockmail_DATA)
+
+EXTRA_DIST = \
+	$(analog_DATA)		\
+	$(calendar_DATA)	\
+	$(classic_DATA)	\
+	$(clock_analog_DATA)	\
+	$(clock_odometer_DATA)	\
+	$(G_DATA)		\
+	$(lcd_DATA)		\
+	$(mailflap_DATA)	\
+	$(fuelgauge_DATA)	\
+	$(counter_DATA)	\
+	$(odometer_DATA)	\
+	$(radar_blue_DATA)	\
+	$(radar_green_DATA)	\
+	$(nice_clock_DATA)	\
+	$(nice_mailcount_DATA)	\
+	$(clock_digital_black_DATA)	\
+	$(clock_digital_red_DATA)	\
+	$(green_analog_digital_DATA)
--- gnome-applets-1.4.0.5/geyes/themes/Makefile.am.orig	Fri Oct 26 11:10:59 2001
+++ gnome-applets-1.4.0.5/geyes/themes/Makefile.am	Fri Sep  6 12:32:21 2002
@@ -1,36 +1,40 @@
 #SUBDIRS = Default Default-tiny Bizarre Bloodshot Horrid
 
 Defaultdir = $(datadir)/geyes/Default
-Default_DATA =
-
-Default_tinydir = $(datadir)/geyes/Default-tiny
-Default_tiny_DATA =
-
-Bizarredir = $(datadir)/geyes/Bizarre
-Bizarre_DATA =
-
-Bloodshotdir = $(datadir)/geyes/Bloodshot
-Bloodshot_DATA =
-
-Horriddir = $(datadir)/geyes/Horrid
-Horrid_DATA =
-
-geyesdir = $(datadir)/geyes
-geyes_DATA = \
+Default_DATA = \
 	Default/config				\
 	Default/Default-eye.png			\
-	Default/Default-pupil.png		\
+	Default/Default-pupil.png
+
+Default_tinydir = $(datadir)/geyes/Default-tiny
+Default_tiny_DATA = \
 	Default-tiny/config			\
 	Default-tiny/Default-tiny-eye.png	\
-	Default-tiny/Default-tiny-pupil.png	\
+	Default-tiny/Default-tiny-pupil.png
+
+Bizarredir = $(datadir)/geyes/Bizarre
+Bizarre_DATA = \
 	Bizarre/config				\
 	Bizarre/Bizarre-eye.png			\
-	Bizarre/Bizarre-pupil.png		\
+	Bizarre/Bizarre-pupil.png
+
+Bloodshotdir = $(datadir)/geyes/Bloodshot
+Bloodshot_DATA = \
 	Bloodshot/config			\
 	Bloodshot/Bloodshot-eye.png		\
-	Bloodshot/Bloodshot-pupil.png		\
+	Bloodshot/Bloodshot-pupil.png
+
+Horriddir = $(datadir)/geyes/Horrid
+Horrid_DATA = \
 	Horrid/config				\
 	Horrid/horrid_eye.png			\
 	Horrid/horrid_pupil.png
 
-EXTRA_DIST = $(geyes_DATA)
\ No newline at end of file
+geyesdir = $(datadir)/geyes
+
+EXTRA_DIST = \
+	$(Default_DATA) \
+	$(Default_tiny_DATA) \
+	$(Bizarre_DATA) \
+	$(Bloodshot_DATA) \
+	$(Horrid_DATA)
--- gnome-applets-1.4.0.5/odometer/themes/Makefile.am.orig	Fri Oct 26 11:12:18 2001
+++ gnome-applets-1.4.0.5/odometer/themes/Makefile.am	Fri Sep  6 12:33:43 2002
@@ -1,49 +1,43 @@
 Chevydir = $(datadir)/odometer/57chevy
-Chevy_DATA =
-chevy_stuff = \
+Chevy_DATA = \
 	57chevy/themedata		\
 	57chevy/57chevy-small-black.png	\
 	57chevy/57chevy-small-red.png
 
 Adir = $(datadir)/odometer/A
-A_DATA = 
-a_stuff = \
+A_DATA = \
 	A/themedata	\
 	A/A.png
 
 bbldotgdir = $(datadir)/odometer/bbldotg
-bbldotg_DATA =
-bbldotg_stuff = \
+bbldotg_DATA = \
 	bbldotg/themedata	\
 	bbldotg/bbldotg.png
 
 odir = $(datadir)/odometer/o
-o_DATA =
-o_stuff = \
+o_DATA = \
 	o/themedata	\
 	o/o-black.png	\
 	o/o-red.png
 
 smplstdir = $(datadir)/odometer/smplst
-smplst_DATA =
-smplst_stuff = \
+smplst_DATA = \
 	smplst/themedata	\
 	smplst/smplst.png
 
 defaultdir = $(datadir)/odometer/default
-default_DATA =
-default_stuff = \
+default_DATA = \
 	default/themedata		\
 	default/default-black.png	\
 	default/default-red.png
 
 zodometerdir = $(datadir)/odometer
-zodometer_DATA = \
-	$(chevy_stuff)		\
-	$(a_stuff)		\
-	$(bbldotg_stuff)	\
-	$(o_stuff)		\
-	$(smplst_stuff)		\
-	$(default_stuff)
 
-EXTRA_DIST = $(zodometer_DATA)
+EXTRA_DIST = \
+	$(Chevy_DATA)		\
+	$(A_DATA)		\
+	$(bbldotg_DATA)		\
+	$(o_DATA)		\
+	$(smplst_DATA)		\
+	$(default_DATA)
+
