diff -Nru gnome-applets-2.0.3.orig/gkb-new/help/C/gkb-C.omf gnome-applets-2.0.3/gkb-new/help/C/gkb-C.omf
--- gnome-applets-2.0.3.orig/gkb-new/help/C/gkb-C.omf	Fri Sep 13 21:59:00 2002
+++ gnome-applets-2.0.3/gkb-new/help/C/gkb-C.omf	Thu Oct 24 10:19:46 2002
@@ -1,6 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" 
-"http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
     <creator>
