diff -Nur gnome-applets-2.0.1/charpick/charpick.c gnome-applets-2.0.1.new/charpick/charpick.c
--- gnome-applets-2.0.1/charpick/charpick.c	Sun Jul 21 17:23:45 2002
+++ gnome-applets-2.0.1.new/charpick/charpick.c	Thu Aug 15 16:45:25 2002
@@ -244,8 +244,6 @@
                break;
     case '$' : p_curr_data->charlist = currency_list;
                break;
-    case '£' : p_curr_data->charlist = currency_list;
-               break;
     case '¤' : p_curr_data->charlist = currency_list;
                break;
     case '1' : p_curr_data->charlist = one_list;
