diff -urN aa/gnome-applets-2.6.0/configure.in gnome-applets-2.6.0/configure.in
--- aa/gnome-applets-2.6.0/configure.in	2004-03-21 20:02:28.000000000 +0100
+++ gnome-applets-2.6.0/configure.in	2004-04-10 17:46:08.224636640 +0200
@@ -208,8 +208,6 @@
 NEED_LIBAPM=no
 build_battstat_applet=yes
 case "${host}" in
-  powerpc-*-linux*)
-	;;
   # list of supported OS cores that use libapm
   *-*-linux*)
 	NEED_LIBAPM=yes
