--- gnome-applets-1.4.0.5.old/cpumemusage/docs/ru/cpumemusage_applet-ru.omf	Thu Jan  1 01:00:00 1970
+++ gnome-applets-1.4.0.5/cpumemusage/docs/ru/cpumemusage_applet-ru.omf	Sun May 19 15:31:47 2002
@@ -0,0 +1,14 @@
+<?xml version="1.0" standalone="no"?>
+<omf>
+  <resource>
+    <title>
+      Индикатор использования ЦПУ и памяти
+    </title>
+    <subject>
+      <category>GNOME|Applets|Monitors</category>
+    </subject>
+    <format mime="text/sgml"/>
+    <identifier url="cpumemusage_applet.sgml"/>
+    <language code="it"/>
+  </resource>
+</omf>
--- gnome-applets-1.4.0.5.old/cpumemusage/docs/ru/Makefile.am	Sun May 19 00:23:26 2002
+++ gnome-applets-1.4.0.5/cpumemusage/docs/ru/Makefile.am	Sun May 19 00:23:26 2002
@@ -1,5 +1,8 @@
-applet = cpumemusage
+figs = \
+	figures/cpumemusage-applet.png
+docname = cpumemusage_applet
 lang = ru
-sgml_ents = $(applet).sgml
-include $(top_srcdir)/applet-docs.make
+omffile = cpumemusage_applet-ru.omf
+sgml_ents = cpumemusage.sgml
+include $(top_srcdir)/sgmldocs.make
 dist-hook: applet-dist-hook
