diff -urN gnome-applets-2.3.1.orig/gweather/docs/Makefile.am gnome-applets-2.3.1/gweather/docs/Makefile.am
--- gnome-applets-2.3.1.orig/gweather/docs/Makefile.am	Mon Apr  7 08:35:21 2003
+++ gnome-applets-2.3.1/gweather/docs/Makefile.am	Mon Apr  7 08:47:26 2003
@@ -1,4 +1,4 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = C de es fr it sv ja ko zh_CN zh_TW
+SUBDIRS = C de es fr sv ja ko zh_CN zh_TW
 
diff -urN gnome-applets-2.3.1.orig/mixer/docs/Makefile.am gnome-applets-2.3.1/mixer/docs/Makefile.am
--- gnome-applets-2.3.1.orig/mixer/docs/Makefile.am	Mon Apr  7 08:35:20 2003
+++ gnome-applets-2.3.1/mixer/docs/Makefile.am	Mon Apr  7 08:35:40 2003
@@ -1 +1 @@
-SUBDIRS = C de es fr it sv ja ko zh_CN zh_TW
+SUBDIRS = C de es fr sv ja ko zh_CN zh_TW
