diff -urN gnome-applets-1.3.1.org/applet-docs.make gnome-applets-1.3.1/applet-docs.make
--- gnome-applets-1.3.1.org/applet-docs.make	Wed Feb 28 12:16:21 2001
+++ gnome-applets-1.3.1/applet-docs.make	Wed Feb 28 12:16:53 2001
@@ -56,7 +56,7 @@
 
 install-data-am: index.html omf
 	-$(mkinstalldirs) $(DESTDIR)$(helpdir)/stylesheet-images
-	-cp $(srcdir)/topic.dat $(DESTDIR)$(helpdir)
+	-cp $(srcdir)/topic.dat $(DESTDIR)$(docdir)
 	-cp $(srcdir)/$(sgml_files) $(DESTDIR)$(helpdir)
 	-for file in \
 		$(applet)_applet/*.html	\

