diff -urN gnome-applets-2.3.0.orig/configure.in gnome-applets-2.3.0/configure.in
--- gnome-applets-2.3.0.orig/configure.in	Thu Mar 27 20:58:30 2003
+++ gnome-applets-2.3.0/configure.in	Thu Mar 27 21:12:22 2003
@@ -15,7 +15,7 @@
 X_LIBS="$X_LIBS $X_PRE_LIBS -lX11 $X_EXTRA_LIBS"
 AC_SUBST(X_LIBS)
 
-GNOME_COMPILE_WARNINGS(yes)
+GNOME_COMPILE_WARNINGS
 
 dnl Stuff we might need, not quite sure yet
 
diff -urN gnome-applets-2.3.0.orig/sgmldocs.make gnome-applets-2.3.0/sgmldocs.make
--- gnome-applets-2.3.0.orig/sgmldocs.make	Thu Mar 27 20:58:30 2003
+++ gnome-applets-2.3.0/sgmldocs.make	Thu Mar 27 21:02:01 2003
@@ -58,7 +58,7 @@
 	-cp $(docname)/index.html .
 
 $(docname).sgml: $(sgml_ents)
-        -ourdir=`pwd`;  \
+        ourdir=`pwd`;  \
         cd $(srcdir);   \
         cp $(sgml_ents) $$ourdir
 
diff -urN gnome-applets-2.3.0.orig/xmldocs.make gnome-applets-2.3.0/xmldocs.make
--- gnome-applets-2.3.0.orig/xmldocs.make	Thu Mar 27 20:58:30 2003
+++ gnome-applets-2.3.0/xmldocs.make	Thu Mar 27 21:02:14 2003
@@ -53,7 +53,7 @@
 	touch omf_timestamp
 
 $(docname).xml: $(entities)
-        -ourdir=`pwd`;  \
+        ourdir=`pwd`;  \
         cd $(srcdir);   \
         cp $(entities) $$ourdir
 
